lib/vector-core/src/stream/driver.rs

```diff
@@ -168,7 +168,7 @@ where
     ) {
         match result {
             Err(error) => {
-                error!(message = "Service call failed.", ?error, request_id);
+                emit(service::CallError { error, request_id });
```

Comment:
Does this error constitute DroppedEvents ?
It seemed like it might, but in inspecting the code, I wasn't sure where the knowledge of the count of dropped events would come from?

Reply:
I have an idea on how to handle this, but it would add on code to this PR. I'll submit this as-is , and open a new PR to address the DroppedEvents.